packages/trader/src/AppV2/Components/ContractCard/contract-card.tsx

```diff
@@ -78,7 +78,7 @@ const ContractCard = ({
     const validToSell = isValidToSell(contractInfo as TContractInfo) && !isSellRequested;
     const isCancelButtonPressed = isSellRequested && isCanceling;
     const isCloseButtonPressed = isSellRequested && isClosing;
-    const Component = redirectTo ? BinaryLink : 'div';
+    const Component = redirectTo ? NavLink : 'div';
```

Comment:
`BinaryLink` can accept prop `to` only as a `string`, it's not possible to pass additional flags like to `NavLink`. That is why I replaced `BinaryLink` ---> `NavLink` + in current production in reports we are also using `NavLink`.
If you want to preserve usage of `BinaryLink`, then refactoring is needed with sanity check of all places where it's used.